core/src/test/java/org/apache/iceberg/rest/TestRESTCatalog.java

```diff
@@ -55,7 +64,19 @@ public void createCatalog() {
 
     this.restCatalog = new RESTCatalog((config) -> adaptor);
     restCatalog.setConf(conf);
-    restCatalog.initialize("prod", ImmutableMap.of());
+
+    restCatalog.initialize("prod", ImmutableMap.of(CatalogProperties.URI, "ignored"));
+  }
+
+  @AfterEach
+  public void closeCatalog() throws IOException {
+    if (restCatalog != null) {
+      restCatalog.close();
+    }
+
+    if (backendCatalog != null) {
+      this.backendCatalog.close();
```

Comment:
Nit: unnecessary `this.` prefix. We only require it when assigning to an instance field.

Reply:
Good catch.